eftest/src/eftest/runner.clj

```diff
@@ -69,10 +78,9 @@
                           (wrap-test-with-timer test-warn-time))]
     (once-fixtures
       (fn []
-        (if (:multithread? opts true)
-          (let [test (bound-fn* test-var)]
-            (dorun (->> vars (filter synchronized?) (map test)))
-            (dorun (->> vars (remove synchronized?) (map (fn [v] #(test v))) run-in-parallel)))
+        (when (:multithread? opts true)
```

Comment:
You've changed the `if` to a `when`, so instead of executing one branch or the other, it executes *both* branches.

Reply:
my fault :( 
i didn't notice, i made change after midnight.
Thanks for review, will fix tomorrow